cocos/animation/marionette/graph-eval.ts

```diff
@@ -91,16 +91,11 @@ export class AnimationGraphEval {
 
         poseLayoutMaintainer.startBind();
 
-        const layerEvaluationRecords = graph.layers.map((layer) => {
-            const record = createLayerEvaluationRecord(
-                layer,
-                bindingContext,
-                clipOverrides,
-            );
-            return record;
-        });
-
-        this._rootPoseNode = new DefaultTopLevelPoseNode(layerEvaluationRecords);
+        this._rootPoseNode = new DefaultTopLevelPoseNode(
```

Comment:
Same as change made to evaluation context. Let `DefaultTopLevelPoseNode` create layer evaluation records instead.